core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/HealthMonitor.scala

```diff
@@ -215,8 +215,8 @@ class HealthMonitor private (val slickDataSource: SlickDataSource,
     // Note: call to `foldMap` depends on SubsystemStatusMonoid
     bucketsToCheck.toList.foldMap { bucket =>
       googleServicesDAO.getBucket(bucket, None).map {
-        case Some(_) => OkStatus
-        case None => failedStatus(s"Could not find bucket: $bucket")
+        case Right(_) => OkStatus
```

Comment:
Should the return type be changed? I'm sorry, but I think I'm still not understanding this Right vs. Left thing.

Reply:
The problem I had before was that the `Some` vs `None` was losing a lot of context. The `Either` allows me to carry around the context as a "wrong" (or `Left`) string rather than just the "didn't work" of the `None`